content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -170,27 +170,27 @@ In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not
    different than {{< param "version" >}}, replace {{< param "version" >}} with
    the desired minor version in the command below.
 
-```bash
-# This overwrites any existing configuration in /etc/yum.repos.d/kubernetes.repo
-cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo
-[kubernetes]
-name=Kubernetes
-baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
-enabled=1
-gpgcheck=1
-gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
-EOF
-```
+   ```bash
+   # This overwrites any existing configuration in /etc/yum.repos.d/kubernetes.repo
+   cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo
+   [kubernetes]
+   name=Kubernetes
+   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
+   enabled=1
+   gpgcheck=1
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   EOF
+   ```
 
 {{< note >}}
 To upgrade kubectl to another minor release, you'll need to bump the version in `/etc/yum.repos.d/kubernetes.repo` before running `yum update`. This procedure is described in more detail in [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
 {{< /note >}}
 
-1. Install kubectl using `yum`:
+2. Install kubectl using `yum`:
 
-```bash
-sudo yum install -y kubectl
-```
+   ```bash
+   sudo yum install -y kubectl
```

Comment:
`yum` should be replaced by `dnf` for newer versions

Reply:
This PR's aim (“fix page layout”) doesn't cover a switch from `yum` to `dnf`. Let's not make that change in _this_ PR.